_pages/CalcNextCommPeriod.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: CalcNextCommPeriod routine
```

Comment:
Titles should be as routines are called in code, so people can easily find them ```calc_next_comm_period```. I also added a filter to directly link to searchresults for the code, but i can add that myself ;-)

Reply:
Fixed